### ISSUE TITLE
(A11y severity 3) Fix accessibility issues in lhnav action modals

### ISSUE THREAD

**Issue by sathomas** [oaeproject/3akai-ux#3945]
While the modals accessed by the file options buttons respond as should be expected, some specific scenarios cause loss of keyboard focus.
- When deleting a document the modal closes causing keyboard focus to revert back to the top of the page. Set keyboard focus to a logical place on the page to maintain keyboard focus (e.g., The "Content" tab).
- When attempting to delete a document and an error appears (e.g., "Document not Deleted: The document could not be deleted"), keyboard focus is lost and reverts back to the top of the page. Set keyboard focus to the page at a logical location when errors occur.
